.docker/run-docker-tests.sh

```diff
@@ -32,9 +32,6 @@ do
 done
 echo ""
 
-# Install needed packages
-/usr/src/scripts/package_pip_packages.sh
-
 pushd /usr/src
```

Comment:
Oh haven't remembered that they are installed here already. Obviously with problems. Do you know why?

Anyway the solution we have now is more proper I guess.

Reply:
No this one was just a test from me. I think was not working as the path was wrong in the container